Makefile.am

```diff
@@ -181,3 +182,12 @@ endif
 if ENABLE_MODULE_RECOVERY
 include src/modules/recovery/Makefile.am.include
 endif
+
+
+if ENABLE_RUST_NAIVETESTS
+$(ECC_SECP):
+	git clone https://github.com/elichai/ecc-secp256k1.git -b v0.1.0 2> /dev/null | true
```

Comment:
Makefile needs to check that this matches a fixed hash. Otherwise malicious code can be easily added to your repo which would then be run on our machines.

nit: ./configure could check if git and cargo are available.

Reply:
hmm so you suggest to hash the whole directory?

Reply:
I'll look into checking that git and cargo are available, i'm pretty new to autotools so this was mostly me shooting in the dark :)

Reply:
Maybe it would be simpler to just vendor the library, i.e., copy the source code into a subfolder.